draft-ietf-quic-qpack.md

```diff
@@ -830,10 +831,8 @@ modifying it.
 
 Each header block is prefixed with two integers.  The Required Insert Count is
 encoded as an integer with an 8-bit prefix after the encoding described in
-{{ric}}).  The Base is encoded as sign-and-modulus integer, using a single sign
```

Comment:
A cursory web search shows that "sign-and-magnitude" is used a lot more often than "sign-and-modulus", which might conjure images of modulo arithmetic.  I propose not to put a name on this encoding, since there's an off-by-one that's not common for sign-and-modulus representations anyway.